{
   "active_lock_reason" : "off-topic",
   "assignee" : null,
   "assignees" : [],
   "author_association" : "NONE",
   "body" : "I have a Windows Server that acts as a full node where other clients from PC connects and synchronize with it, all is ok, then I decided to add another hosted vps based on ubuntu 18.04, it doesn't synchronize even it seems number of blocks, I opened the tcp port with ufw but I don't know why it's not syncing:\r\n\r\n```\r\n[2021-01-21T17:11:35Z testcoin Core version v1.0.0.0-g (release build)\r\n2021-01-21T17:11:35Z InitParameterInteraction: parameter interaction: -connect set -> setting -dnsseed=0\r\n2021-01-21T17:11:35Z InitParameterInteraction: parameter interaction: -connect set -> setting -listen=0\r\n2021-01-21T17:11:35Z InitParameterInteraction: parameter interaction: -listen=0 -> setting -upnp=0\r\n2021-01-21T17:11:35Z InitParameterInteraction: parameter interaction: -listen=0 -> setting -discover=0\r\n2021-01-21T17:11:35Z InitParameterInteraction: parameter interaction: -listen=0 -> setting -listenonion=0\r\n2021-01-21T17:11:35Z InitParameterInteraction: parameter interaction: -whitelistforcerelay=1 -> setting -whitelistrelay=1\r\n2021-01-21T17:11:35Z Assuming ancestors of block 00000a49e6b0d232e39d3c20ed7f9000e034047d42370e869270ec7cb595d6ec have valid signatures.\r\n2021-01-21T17:11:35Z Setting nMinimumChainWork=0000000000000000000000000000000000000000000000000000000000200020\r\n2021-01-21T17:11:35Z Using the 'sse4(1way),sse41(4way)' SHA256 implementation\r\n2021-01-21T17:11:35Z Using RdRand as an additional entropy source\r\n2021-01-21T17:11:35Z Default data directory /root/.testcoin\r\n2021-01-21T17:11:35Z Using data directory /root/.testcoin\r\n2021-01-21T17:11:35Z Using config file /root/.testcoin/testcoin.conf\r\n2021-01-21T17:11:35Z Using at most 125 automatic connections (1024 file descriptors available)\r\n2021-01-21T17:11:35Z Using 16 MiB out of 32/2 requested for signature cache, able to store 524288 elements\r\n2021-01-21T17:11:35Z Using 16 MiB out of 32/2 requested for script execution cache, able to store 524288 elements\r\n2021-01-21T17:11:35Z Using 0 threads for script verification\r\n2021-01-21T17:11:35Z HTTP: creating work queue of depth 16\r\n2021-01-21T17:11:35Z No rpcpassword set - using random cookie authentication.\r\n2021-01-21T17:11:35Z Generated RPC authentication cookie /root/.testcoin/.cookie\r\n2021-01-21T17:11:35Z HTTP: starting 4 worker threads\r\n2021-01-21T17:11:35Z Using wallet directory /root/.testcoin\r\n2021-01-21T17:11:35Z init message: Verifying wallet(s)...\r\n2021-01-21T17:11:35Z Using BerkeleyDB version Berkeley DB 4.8.30: (April  9, 2010)\r\n2021-01-21T17:11:35Z Using wallet wallet.dat\r\n2021-01-21T17:11:35Z BerkeleyEnvironment::Open: LogDir=/root/.testcoin/database ErrorFile=/root/.testcoin/db.log\r\n2021-01-21T17:11:35Z scheduler thread start\r\n2021-01-21T17:11:35Z Cache configuration:\r\n2021-01-21T17:11:35Z * Using 2.0MiB for block index database\r\n2021-01-21T17:11:35Z * Using 8.0MiB for chain state database\r\n2021-01-21T17:11:35Z * Using 440.0MiB for in-memory UTXO set (plus up to 286.1MiB of unused mempool space)\r\n2021-01-21T17:11:35Z init message: Loading block index...\r\n2021-01-21T17:11:35Z Opening LevelDB in /root/.testcoin/blocks/index\r\n2021-01-21T17:11:35Z Opened LevelDB successfully\r\n2021-01-21T17:11:35Z Using obfuscation key for /root/.testcoin/blocks/index: 0000000000000000\r\n2021-01-21T17:11:35Z LoadBlockIndexDB: last block file = 0\r\n2021-01-21T17:11:35Z LoadBlockIndexDB: last block file info: CBlockFileInfo(blocks=1, size=288, heights=0...0, time=2021-01-10...2021-01-10)\r\n2021-01-21T17:11:35Z Checking all blk files are present...\r\n2021-01-21T17:11:35Z Opening LevelDB in /root/.testcoin/chainstate\r\n2021-01-21T17:11:35Z Opened LevelDB successfully\r\n2021-01-21T17:11:35Z Using obfuscation key for /root/.testcoin/chainstate: fec815e2991463fd\r\n2021-01-21T17:11:35Z Loaded best chain: hashBestChain=00000a49e6b0d232e39d3c20ed7f9000e034047d42370e869270ec7cb595d6ec height=0 date=2021-01-10T10:15:11Z progress=1.000000\r\n2021-01-21T17:11:35Z init message: Rewinding blocks...\r\n2021-01-21T17:11:35Z init message: Verifying blocks...\r\n2021-01-21T17:11:35Z  block index              12ms\r\n2021-01-21T17:11:35Z init message: Loading wallet...\r\n2021-01-21T17:11:35Z [default wallet] nFileVersion = 1000000\r\n2021-01-21T17:11:35Z [default wallet] Keys: 2001 plaintext, 0 encrypted, 2001 w/ metadata, 2001 total. Unknown wallet records: 1\r\n2021-01-21T17:11:35Z [default wallet] Wallet completed loading in              60ms\r\n2021-01-21T17:11:35Z [default wallet] setKeyPool.size() = 2000\r\n2021-01-21T17:11:35Z [default wallet] mapWallet.size() = 0\r\n2021-01-21T17:11:35Z [default wallet] mapAddressBook.size() = 0\r\n2021-01-21T17:11:35Z mapBlockIndex.size() = 6\r\n2021-01-21T17:11:35Z nBestHeight = 0\r\n2021-01-21T17:11:35Z init message: Loading P2P addresses...\r\n2021-01-21T17:11:35Z Loaded 3 addresses from peers.dat  0ms\r\n2021-01-21T17:11:35Z init message: Loading banlist...\r\n2021-01-21T17:11:35Z init message: Starting network threads...\r\n2021-01-21T17:11:35Z DNS seeding disabled\r\n2021-01-21T17:11:35Z init message: Done loading\r\n2021-01-21T17:11:35Z msghand thread start\r\n2021-01-21T17:11:35Z opencon thread start\r\n2021-01-21T17:11:35Z addcon thread start\r\n2021-01-21T17:11:35Z Imported mempool transactions from disk: 0 succeeded, 0 failed, 0 expired, 0 already there\r\n2021-01-21T17:11:35Z net thread start\r\n2021-01-21T17:11:35Z New outbound peer connected: version: 70015, blocks=202, peer=0\r\n2021-01-21T17:11:35Z ERROR: AcceptBlockHeader: Consensus::ContextualCheckBlockHeader: 0000015584b1352a512e2139846c49d8ecd408d7d945c5651e7036b8ab846198, bad-diffbits, incorrect proof of work (code 16)\r\n2021-01-21T17:11:35Z Warning: not punishing manually-connected peer 35.238.12.61:11890!\r\n```\r\n\r\n\r\nIf you checked the following line you would conclude that it sees 202 blocks, so how come on earth it doesn't sync?\r\n\r\n`2021-01-21T17:11:35Z New outbound peer connected: version: 70015, blocks=202, peer=0`",
   "closed_at" : "2021-01-21T18:21:12Z",
   "closed_by" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/6399679?v=4",
      "events_url" : "https://api.github.com/users/MarcoFalke/events{/privacy}",
      "followers_url" : "https://api.github.com/users/MarcoFalke/followers",
      "following_url" : "https://api.github.com/users/MarcoFalke/following{/other_user}",
      "gists_url" : "https://api.github.com/users/MarcoFalke/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/MarcoFalke",
      "id" : 6399679,
      "login" : "MarcoFalke",
      "node_id" : "MDQ6VXNlcjYzOTk2Nzk=",
      "organizations_url" : "https://api.github.com/users/MarcoFalke/orgs",
      "received_events_url" : "https://api.github.com/users/MarcoFalke/received_events",
      "repos_url" : "https://api.github.com/users/MarcoFalke/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/MarcoFalke/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/MarcoFalke/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/MarcoFalke"
   },
   "comments" : 1,
   "comments_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/20979/comments",
   "created_at" : "2021-01-21T18:17:21Z",
   "events_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/20979/events",
   "html_url" : "https://github.com/bitcoin/bitcoin/issues/20979",
   "id" : 791387342,
   "labels" : [
      {
         "color" : "FBBAAB",
         "default" : false,
         "description" : null,
         "id" : 64585,
         "name" : "Bug",
         "node_id" : "MDU6TGFiZWw2NDU4NQ==",
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/labels/Bug"
      }
   ],
   "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/20979/labels{/name}",
   "locked" : true,
   "milestone" : null,
   "node_id" : "MDU6SXNzdWU3OTEzODczNDI=",
   "number" : 20979,
   "performed_via_github_app" : null,
   "repository_url" : "https://api.github.com/repos/bitcoin/bitcoin",
   "state" : "closed",
   "title" : "Issue synchronizing with the blockchain",
   "updated_at" : "2021-01-21T18:21:22Z",
   "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/20979",
   "user" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/71932943?v=4",
      "events_url" : "https://api.github.com/users/webdev86/events{/privacy}",
      "followers_url" : "https://api.github.com/users/webdev86/followers",
      "following_url" : "https://api.github.com/users/webdev86/following{/other_user}",
      "gists_url" : "https://api.github.com/users/webdev86/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/webdev86",
      "id" : 71932943,
      "login" : "webdev86",
      "node_id" : "MDQ6VXNlcjcxOTMyOTQz",
      "organizations_url" : "https://api.github.com/users/webdev86/orgs",
      "received_events_url" : "https://api.github.com/users/webdev86/received_events",
      "repos_url" : "https://api.github.com/users/webdev86/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/webdev86/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/webdev86/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/webdev86"
   }
}
